src/core/execute.c

```diff
@@ -749,11 +749,18 @@ static ConfirmAnswer ask_for_confirmation(const char *vc, const char *cmdline) {
 
                 switch (c) {
                 case '?':
-                        printf("  y - yes, execute the command\n"
+                        printf("  i - info, show a short summary of the unit\n"
+                               "  y - yes, execute the command\n"
                                "  f - fail, don't execute the command and pretend it failed\n"
                                "  s - skip, don't execute the command and pretend it succeeded\n"
                                "  ? - help\n");
-                        continue;
+                        continue;  /* ask again */
+                case 'i':
+                        printf("  Description: %s\n"
+                               "  Unit:    %s\n"
+                               "  Command: %s\n",
```

Comment:
indenting is strange: you add extra whitespace to the second line, but only to match the third like. It would appear more natural to me to either add enough whitespace so that all lines align, or add no no whitespace at all...